### DISCUSSION TITLE
Welcome to zsmq Discussions!

### DISCUSSION THREAD

**Discussion by dhslrl321** [dhslrl321/zsmq#24] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tgsong827** [comment]
Hi, I'm new in zsmq.

**Reply by dhslrl321** [reply]
Hi~ welcome bro
